test/mpi/threads/testlist.in

```diff
@@ -5,3 +5,4 @@ mpi_t
 @spawndir@
 @rmadir@
 coll
+perf
```

Comment:
Let's remove this for now. We'll only run this test manually or with custom tests for now.

Reply:
Removed this in the latest version.